lib/multi-acc-new-eks-mixed-observability-pattern/grafana-operator-index.ts

```diff
@@ -0,0 +1,45 @@
+import { Construct } from 'constructs';
```

Comment:
rename this file to something meaning ful like monitoring account or something

Reply:
done. renamed to central-monitoring.ts